src/site/_data/authorsData.json

```diff
@@ -1574,5 +1574,20 @@
       "given": "Matt",
       "family": "Gaunt"
     }
+  },
+  "bingler": {
+    "name": {
+      "given": "Steven",
+      "family": "Bingler"
+    },
+    "org": {
+      "name": "Google",
+      "unit": "Chrome"
+    },
+    "descriptions": {
+      "en": "Software engineer on Chrome's improvements to anti-covert tracking efforts."
```

Comment:
Looks like this is a bit out dated, can you change to
"Software engineer on Chrome's Privacy Sandbox."

Reply:
Apologies, missed that in pulling from the draft!

Reply:
LGTM now, thanks!